src/components/ApplicationDashboard.js

```diff
@@ -230,7 +232,14 @@ const Dashboard = ({
       <StatusContainer>
         <div>
           <AppStatusText>
-            Application status: {hackerStatuses(relevantDates, username)[hackerStatus]?.cardText}
+            Application status:{' '}
+            {hackerStatus === 'acceptedNoResponseYet'
+              ? !isLoadingAppStatus
+                ? isRsvpOpen
+                  ? hackerStatuses()[hackerStatus]?.cardText
+                  : 'NO RSVP'
+                : null
+              : hackerStatuses()[hackerStatus]?.cardText}
```

Comment:
<img width="1494" alt="Screen Shot 2020-12-30 at 4 43 28 PM" src="https://user-images.githubusercontent.com/38872354/103388584-a3f64700-4abe-11eb-93f5-ac98da7086b6.png">
<img width="1576" alt="Screen Shot 2020-12-30 at 4 47 37 PM" src="https://user-images.githubusercontent.com/38872354/103388602-bd978e80-4abe-11eb-8196-006dbb144e5b.png">

The message for No RSVP should be the same as that of un-RSVP, except instead of `un-RSVP'd`, it should say `No RSVP` - can you make this change? 
 
Also for code quality and better separation of concerns, it would be better if this logic was handled in `hackerStatuses` - i.e. in the parent container, have logic to set the status to `acceptedNoRSVP`, then add another case for `acceptedNoRSVP` in `hackerStatuses` and rename `acceptedNotAttending`.

```suggestion
            Application status: {hackerStatuses()[hackerStatus]?.cardText}
```